cmd/activator/main.go

```diff
@@ -123,6 +124,13 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// Report stats on Go memory usage every 30 seconds.
+	msp := metrics.NewMemStatsAll()
```

Comment:
This looks like it should be exported by the `metrics` itself. It's repeated the same way (modulo which logger to use).

Reply:
The place we do this for other metrics is in `knative.dev/pkg/controller` in an `init()`.  Doing it in `controller` felt misplaced, introducing an `init()` in metrics felt aggressive, introducing an `_ "knative.dev/pkg/metrics/gomemstats"` felt like overkill, and unless we `log.Fatal` in the hoisted boilerplate function this saves 1 line.

I think if I were going to do anything it would be the `_` case as sugar for this, but that'll need another knative/pkg dance.

Now that I've brain dumped, What did you have in mind? :)

Reply:
I though jut expose it as a function in `metrics` and call in each binary, i.e. 6->1 line conversion.